libr/cons/grep.c

```diff
@@ -275,7 +275,11 @@ static int cmp(const void *a, const void *b) {
 	const char *ca = r_str_chop_ro (a);
 	const char *cb = r_str_chop_ro (b);
 	if (!a || !b) {
+#ifdef _MSC_VER
```

Comment:
remove the non-msc-ver code and drop the ifdef. this code should build fine on non-msvc